internal/provider/resource_consul_cluster_test.go

```diff
@@ -26,10 +26,15 @@ resource "hcp_consul_cluster" "test" {
 data "hcp_consul_cluster" "test" {
 	cluster_id = hcp_consul_cluster.test.cluster_id
 }
+
+resource "hcp_consul_cluster_root_token" "test" {
+	cluster_id = hcp_consul_cluster.test.cluster_id
+}
 `
 
-// This includes tests against both the resource and the corresponding datasource
-// to shorten testing time.
+// This includes tests against both the resource, the corresponding datasource,
+// and creation of the Consul cluster root token resource in order to shorten
+// testing time.
```

Comment:
Could we add some assertions below on the root token output? (`accessor_id`, `secret_id`, and `kubernetes_secret`)

Reply:
Tests results after updates:
```
$ make testacc TESTARGS='-run=TestAccConsulCluster'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/... -v -run=TestAccConsulCluster -timeout 120m
?   	github.com/hashicorp/terraform-provider-hcp/internal/clients	[no test files]
testing: warning: no tests to run
PASS
ok  	github.com/hashicorp/terraform-provider-hcp/internal/consul	0.692s [no tests to run]
testing: warning: no tests to run
PASS
ok  	github.com/hashicorp/terraform-provider-hcp/internal/input	1.422s [no tests to run]
=== RUN   TestAccConsulCluster
--- PASS: TestAccConsulCluster (582.54s)
PASS
ok  	github.com/hashicorp/terraform-provider-hcp/internal/provider	583.039s
```